src/mame/taito/taitopjc.cpp

```diff
@@ -947,4 +947,4 @@ ROM_END
 } // anonymous namespace
 
 
-GAME( 1998, optiger, 0, taitopjc, taitopjc, taitopjc_state, init_optiger, ROT0, "Taito", "Operation Tiger (Ver 2.14 O)", MACHINE_IMPERFECT_GRAPHICS | MACHINE_IMPERFECT_TIMING | MACHINE_NO_SOUND )
+GAME( 1998, optiger, 0, taitopjc, taitopjc, taitopjc_state, init_optiger, ROT0, "Taito", "Operation Tiger (Ver 2.14 O)", MACHINE_NOT_WORKING | MACHINE_IMPERFECT_GRAPHICS | MACHINE_NO_SOUND )
```

Comment:
I didn't say remove the `MACHINE_IMPERFECT_TIMING` tho.

Reply:
You're speaking my language now. Consider this part done.